pilot/pkg/security/authz/builder/testdata/all-fields-out.yaml

```diff
@@ -65,6 +74,9 @@ typedConfig:
                   - header:
                       name: :method
                       presentMatch: true
+                  - header:
+                      exactMatch: regex:not-method-regex
```

Comment:
some of these are still using exact matching?

Reply:
Fixed HeaderMatching. Thanks!